ynput/OpenPype
# Maya: Multipart fix

## Brief description
Fix multipart logic in render products.

## Description
Each renderer has a different way of defining whether output images is multipart, so we need to define it for each renderer. Also before the `multipart` class variable was defined multiple times in several places, which made it tricky to debug where `multipart` was defined. Now its created on initialization and referenced as `self.multipart`

## Testing notes:
1. Create render in Maya with a supported renderer that support multipart image output (EXR).
2. Publish and inspect the produced json to confirm no AOV image outputs are expected.
